<?xml version="1.0"?>
<launch>
  <!-- BEGIN ROBOT 1-->
  <group ns="robot1">
    <param name="tf_prefix" value="robot1" />
    <include file="$(find multi_turtlebot)/launch/one_turtlebot.launch" >
      <arg name="init_pose" value="-x 3 -y 3 -z 0" />
      <arg name="robot_name"  value="robot1" />
      <arg name="initial_pose_x" value="3" />
      <arg name="initial_pose_y" value="3" />
    </include>
  </group>

  <!-- BEGIN ROBOT 2-->
  <group ns="robot2">
    <param name="tf_prefix" value="robot2" />
    <include file="$(find multi_turtlebot)/launch/one_turtlebot.launch" >
      <arg name="init_pose" value="-x 3 -y 4 -z 0" />
      <arg name="robot_name"  value="robot2" />
      <arg name="initial_pose_x" value="3" />
      <arg name="initial_pose_y" value="4" />
    </include>
  </group>
  <!-- BEGIN ROBOT 3-->
  <group ns="robot3">
    <param name="tf_prefix" value="robot3" />
    <include file="$(find multi_turtlebot)/launch/one_turtlebot.launch" >
      <arg name="init_pose" value="-x 3 -y 5 -z 0" />
      <arg name="robot_name"  value="robot3" />
      <arg name="initial_pose_x" value="3" />
      <arg name="initial_pose_y" value="5" />
    </include>
  </group>
</launch>
